src/reducers/inputs.js

```diff
@@ -42,6 +42,31 @@ export const setMyItems = (state, action) => {
   return newState
 }
 
+export const setMyItemsLang = (state, action) => {
+  const newState = { ...state }
+  const reduxPath = action.payload.reduxPath
+  let level = 0
+
+  reduxPath.reduce((obj, key) => {
+    level++
+    if (level === reduxPath.length) {
+      if ((key in obj) != true) {
+        obj[key] = { items: [] }
+      }
+      obj[key].items.forEach(item => {
```

Comment:
I wonder if this should use `map()` instead since you're modifying values in the array?

Reply:
On second thought, no, I don't think `map()` is a good fit here since you're looking to mutate a single property in a single object in the array (I think?).